hack/testing/init-log-stack

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 OS_ANSIBLE_VERBOSITY=${OS_ANSIBLE_VERBOSITY:-"vv"}
-OS_VERSION_STRING=${OS_VERSION:+openshift_release=$OS_VERSION}
+OS_VERSION_STRING="openshift_release=v1.5.0"
```

Comment:
I don't know. I assume since we are using the same job it will come in as an env var like '3.6' but I didnt want to take that chance.